docs/source/examples/cnn.rst

```diff
@@ -36,6 +36,9 @@ LeNet5
 ''''''
 
 Here, let's start by defining LeNet5 [LeCun98]_ in Chainer.
+In this example, we show 
+`a simplified version of LeNet5 <http://deeplearning.net/tutorial/lenet.html#lenet>`_
+that is explained in DeepLearning 0.1 documentation.
```

Comment:
I'm wondering if the name "DeepLearning 0.1 documentation" is good for referring the site. "Deep Learning Tutorials" may be better.
```suggestion
introduced in Deep Learning Tutorials.
```